exec/java-exec/src/test/java/org/apache/drill/exec/TestSSLConfig.java

```diff
@@ -49,12 +64,19 @@ public void testMissingKeystorePassword() throws Exception {
     ConfigBuilder config = new ConfigBuilder();
     config.put(ExecConstants.HTTP_KEYSTORE_PATH, "/root");
     config.put(ExecConstants.HTTP_KEYSTORE_PASSWORD, "");
+    config.put(ExecConstants.SSL_USE_HADOOP_CONF, false);
+    config.put(ExecConstants.USER_SSL_ENABLED, true);
     try {
-      SSLConfig sslv = new SSLConfig(config.build());
+      SSLConfig sslv = new SSLConfigBuilder()
+          .config(config.build())
+          .mode(SSLFactory.Mode.SERVER)
+          .initializeSSLContext(false)
+          .validateKeyStore(true)
+          .build();
       fail();
       //Expected
     } catch (Exception e) {
-      assertTrue(e instanceof DrillException);
+
```

Comment:
So what is test is actually testing? Since we just fail and ignore the exception.